project/Build.scala

```diff
@@ -836,7 +836,10 @@ object Build extends sbt.Build {
       base = file("examples") / "reversi",
       settings = exampleSettings ++ Seq(
           name := "Reversi - Scala.js example",
-          moduleName := "reversi"
+          moduleName := "reversi",
+          scalaJSOutputWrapper := (
+              "// Scala.js - Reversi\n//\n// Compiled with Scala.js\n",
+              "// End of Scala.js generated script")
```

Comment:
This would be a better fit for an actual test in one of the projects of `sbt-plugin-test`, with an automated test that opens the output of fastOptJS and fullOptJS, and verifies that the starts and ends with the appropriate content.